core/src/test/scala/com/nulabinc/backlog/migration/SimpleFixture.scala

```diff
@@ -152,6 +152,7 @@ trait SimpleFixture {
   val numericCustomFieldId    = 3
   val numericCustomFieldName  = "numeric"
   val numericCustomFieldValue = "12.12"
+  val numericCustomFieldScaledValue = "12.1200"
```

Comment:
Add a test value to support the BigDecimal scale

https://github.com/nulab/backlog4j/blob/master/src/main/java/com/nulabinc/backlog4j/api/option/CreateIssueParams.java#L496
https://github.com/nulab/backlog4j/blob/master/src/main/java/com/nulabinc/backlog4j/api/option/UpdateIssueParams.java#L337